### PR TITLE
Make Altoclef 1% less scuffed

### PR DESCRIPTION
**Pull request by KTibow**
- FIx spelling/rewrite some text
- Do some stuff properly to satisfy VS Code

Nothing should change functionality.

### CODE REVIEW

**Review by SnivyLlama — APPROVED** [review]
Amazing work comrade

**Comment by KTibow** [comment]
From a small test, this doesn't seem to break anything.

**Comment by KTibow** [comment]
I tested the build, it works okay for what I can tell.

File: develop.md
Comment (adrisj7) [inline develop.md:31]: Replacing `#)` with `#.` doesn't add much

File: develop.md
Comment (adrisj7) [inline develop.md:5]: This feels like a general tutorial for formatting, while good advice I don't see how it's relevant to altoclef developers themselves.

File: README.md
Comment (adrisj7) [inline README.md:2]: This file has just spaces and one minor format change, not really necessary.

File: .github/workflows/javadoc-publish.yml
Comment (adrisj7) [inline .github/workflows/javadoc-publish.yml:15]: Replacing single quotes with double quote doesn't do much here

File: .github/workflows/gradle.yml
Comment (adrisj7) [inline .github/workflows/gradle.yml:14]: Just a bunch of spaces

File: .github/ISSUE_TEMPLATE/minor_bug_report.md
Comment (adrisj7) [inline .github/ISSUE_TEMPLATE/minor_bug_report.md:4]: Single quote -> double quote doesn't do much

File: .github/ISSUE_TEMPLATE/major_bug_report.md
Comment (adrisj7) [inline .github/ISSUE_TEMPLATE/major_bug_report.md:4]: Single quote -> double quote doesn't do much

File: .github/ISSUE_TEMPLATE/feature-improvement-request.md
Comment (adrisj7) [inline .github/ISSUE_TEMPLATE/feature-improvement-request.md:4]: Single quote -> double quote doesn't do much

File: src/main/java/adris/altoclef/AltoClef.java
Comment (adrisj7) [inline src/main/java/adris/altoclef/AltoClef.java:140]: Lmao what is this

**Review by adrisj7 — CHANGES_REQUESTED** [review]
OK the spelling errors I could get behind, but now it looks like you just ran the repo through a formatter. I wouldn't mind accepting a merge but I don't feel like granting a massive line count contribution would be fair to the people who spent a lot of time adding features. I'll merge this if you remove the auto format changes and just stick with the changes you did yourself

File: src/main/java/adris/altoclef/AltoClef.java
Comment (KTibow) [reply src/main/java/adris/altoclef/AltoClef.java:140]: The code was completely unused.

**Comment by KTibow** [comment]
Alright, I can see how if statistics matters, an auto formatter really messes stuff up. (In this case it also caused 90% of your complaints lol)
I'll attempt to re-add the changes based on the current formatting.

File: .github/ISSUE_TEMPLATE/feature-improvement-request.md
Comment (Bluscream) [reply .github/ISSUE_TEMPLATE/feature-improvement-request.md]: looks nicer tho.

Comment (KTibow) [reply .github/ISSUE_TEMPLATE/feature-improvement-request.md]: eeh i undid the changes by Prettier because the guy who made the whole project didnt like it

**Review by adrisj7 — CHANGES_REQUESTED** [review]
This is good! Only a few minor notes.

If you can't make the changes I described that's fine, I'll add them in before merging when I get back to this project.

File: src/main/java/adris/altoclef/tasks/resources/TradeWithPiglinsTask.java
Comment (KTibow) [reply src/main/java/adris/altoclef/tasks/resources/TradeWithPiglinsTask.java:29]: Well then where is it used?

File: src/main/java/adris/altoclef/util/BlockRange.java
Comment (KTibow) [reply src/main/java/adris/altoclef/util/BlockRange.java:15]: Done.

**Comment by KTibow** [comment]
Stale and conflicting

**Pull request closed without merging**
